config.yaml

```diff
@@ -5,6 +5,9 @@ options:
   cluster-name:
     description: "Optional - Name of the MySQL InnoDB cluster"
     type: "string"
+  cluster-set-domain-name:
+    description: "Optional - Domain name for the MySQL InnoDB cluster set"
+    type: "string"
```

Comment:
Is the name approved by @delgod / Mohamed?
Should be ACKed quickly, but it is a procedure.

Reply:
The direction is to remove the config altogether and use an auto-generated.

Reply:
b147c53